saga/src/env.ts

```diff
@@ -0,0 +1,48 @@
+// Helpers for configuring the mTLS client and worker samples
```

Comment:
@lorensr we should probably reuse this utility in other samples, it would be useful to show users how to set all of this up.

Reply:
Thinking about this some more, I'd rather put this "advanced" setup in a dedicated sample to simplify.

Reply:
by `advanced` you mean the toggle between remote and local connectivity?

Reply:
i definitely agree..removed and isolated in a separate tool